server/store.go

```diff
@@ -123,6 +135,7 @@ func (s *storeInfo) getLocationID(keys []string) string {
 type StoreStatus struct {
 	*pdpb.StoreStats
 
+	Blocked           bool      `json:"blocked"`
```

Comment:
could you think a better name? If can't, please add more comment for the block here, but I still think block is not good here...